### PR TITLE
Dark Mode - Add settings option to select UI colour

### PR DESCRIPTION
**Pull request by petrucianno**
Allow user to change the UI theme from settings dialog
SettingsDialog

### CODE REVIEW

File: src/tablemodel.cpp [hunk 1]
```diff
@@ -508,11 +508,14 @@ QColor TableModel::getMsgBackgroundColor(QDltMsg &msg,int index,long int filterp
...
+    if (QDltSettingsManager::UI_Colour::UI_Dark == QDltSettingsManager::getInstance()->uiColour)
+    {
+        brushColor = QColor(31,31,31);
+    }
     #ifdef Q_OS_WIN
         QSettings themeSettings("HKEY_CURRENT_USER\\Software\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize",QSettings::NativeFormat);
-        if(themeSettings.value("AppsUseLightTheme")==0){
-            brushColor = QColor(31,31,31);
-        }
+        if(themeSettings.value("AppsUseLightTheme")==0){}
```
Comment (eengnr) [inline src/tablemodel.cpp]: Is this still necessary?

**Review by alexmucde — APPROVED** [review]

**Comment by eengnr** [comment]
Very nice PR, I never had time for this extra feature!

File: src/tablemodel.cpp (hunk 1 above)
Comment (petrugrd) [reply src/tablemodel.cpp]: No, just forgot to remove

**Comment by petrucianno** [comment]
@eengnr, thank you. Hope it will be helpful.

**Comment by alexmucde** [comment]
Thanks for the contribution. We will test it and give you feedback if needed. Sorry for the delay.

**Comment by Usha-Rajanna** [comment]
The Scenarios are working fine with Dark Mode and Light mode, only concern is to restart the application after the setting is applied from Dark to Light/ Light to Dark to see the changes.

**Pull request merged**